### ISSUE TITLE
Artificial Neural Networks [ANNs] is it possible ?

### ISSUE THREAD

**Issue by Quetzalquatl** [akumidv/tradingview-assistant-chrome-extension#40]
As I wrote in the title, the annealing method is really nice. It will be much more successful if it can find the right result faster and with less repetitions.
-Also, if you add artificial intelligence to the project, I really get teary-eyed.
-and the project must have a time frame and be scannable.
-Can the right time frame be found first with artificial intelligence or annealing method?
These are the ones that come to my mind now. Thanks in advance. Actually I am a programmer but I know very little javascript.

**Comment by akumidv** [comment]
Neural Networks don't make sense by themselves. You need to create a network architecture, train it with data, and only then it can be used as a strategy. All this requires high resources, but the calculation of strategies in trading view is very limited.
It is not difficult to make the neural network itself on simple neurons in tradingview. The problem is that the NN data needs to be set from somewhere. This can be done potentially, just like you can load external signals in an extension. But the network itself will be small and of a rather rigid architecture, and generally of little use.
Annealing method is one of data scientist methods. I think to add two most popular - genetic algorithm or gradient buster method for optimization. May be Nesterov method. But I think not earlier than in Autumn.

**Comment by Quetzalquatl** [comment]
The Nesterov method and Adam Optimizer are really nice. I hope you add it to the project as soon as possible. Waiting Impatiently. Thank you for your hard work.
